otter/auth.py

```diff
@@ -284,6 +284,42 @@ def __hash__(self):
                      self._url, self._admin_url))
 
 
+@implementer(IAuthenticator)
+class SingleTenantAuthenticator(object):
+    """
+    An authentication handler that authenticates using a single tenant id.
+    """
+    def __init__(self, _identity_user, _identity_password, url):
```

Comment:
Actually, I am sorry didn't notice that auth request is sending `tenant_id` in it. This will ensure that auth on any other tenant fails. Also, the [docs](http://docs.rackspace.com/auth/api/v2.0/auth-client-devguide/content/POST_authenticate_v2.0_tokens_Token_Calls.html) discuss ways where tenant can be useful. So, kindly ignore my concern about passing the `tenant_id` to the constructor :)